Data/Aeson/Types/Instances.hs

```diff
@@ -112,6 +112,7 @@ import Data.Foldable (Foldable)
 import Data.Monoid (mempty)
 import Data.Traversable as Tr (traverse)
 import Data.Word (Word)
+import Numeric.Natural
```

Comment:
This is wrong. Note the `!` before `!MIN_VERSION_base(4,8,0)`. You want to import Numeric.Natural when the minimal version of base is 4.8. So you want this piece to read:

```haskell
#if MIN_VERSION_base(4,8,0)
import Numeric.Natural
#else
import Control.Applicative ((<$>), (<*>), pure)
import Data.Foldable (Foldable)
import Data.Monoid (mempty)
import Data.Traversable as Tr (traverse)
import Data.Word (Word)
#endif
```